<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1953</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>video</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/ladybug.png</normaloff>icon/ladybug.png</iconset>
  </property>
  <widget class="QLabel" name="video_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loadinglabel">
   <property name="geometry">
    <rect>
     <x>1740</x>
     <y>860</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loadinglabel_2">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>860</y>
     <width>550</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/Loding.png</pixmap>
   </property>
  </widget>
  <zorder>video_label</zorder>
  <zorder>loadinglabel_2</zorder>
  <zorder>loadinglabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
